addons/l10n_jo_edi/views/report_invoice.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <template id="l10n_jo_edi_report_invoice_document" inherit_id="account.report_invoice_document">
```

Comment:
Do like the others: put a `primary=True`. That way, you will be able to change the template with anything you need later without facing trouble with the standard template

Reply:
I don't understand the purpose of the `primary="True"` tbh, but when I added it, the QR code disappeared from the PDFs